stable/admin/templates/statefulset.yaml

```diff
@@ -71,6 +72,10 @@ spec:
           valueFrom:
             fieldRef:
               fieldPath: metadata.name
+        - name: POD_UID
+          valueFrom:
+            fieldRef:
+              fieldPath: metadata.uid
```

Comment:
Sorry, I wasn't clear in the summary/commit-message. The POD_UID is unrelated to the RBAC changes, but I thought it would be nice to have. We do not need RBAC in order to use Downward APIs (we already do it to get the Pod name).